packages/vtt/src/hooks/useRenderer.js

```diff
@@ -1,21 +1,32 @@
 // Module imports
-import { useEffect } from 'react'
+import {
+	useEffect,
+	useRef,
+} from 'react'
 
 // Local imports
+import { drawGrid } from 'helpers/drawGrid'
 import { scaleCanvas } from 'helpers/scaleCanvas'
 
+function createCanvas() {
+	if (typeof window === 'undefined') {
+		return null
+	}
+
+	return document.createElement('canvas')
+}
+
 export function useRenderer(canvasRef, dependencies) {
+	const gridRef = useRef(createCanvas())
+
 	return useEffect(() => {
 		const canvasElement = canvasRef.current
+		const gridElement = gridRef.current
 
-		if (canvasElement) {
-			const {
-				height,
-				width,
-			} = canvasElement
+		if ((typeof window !== 'undefined') && canvasElement) {
 			const context = canvasElement.getContext('2d')
 
-			scaleCanvas(canvasElement)
+			scaleCanvas(canvasElement, gridElement)
 
 			let frame = 0
```

Comment:
frame is defined outside the loop that it is being incremented in via let which I don't think make sense since let is locally-scoped